cpp/test/build/compare_yk_sep.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash -x
+
+# Run test_lw and grep for sentinel string that report values
+# for arrays/views you are having problems with. To use this
+# feature, add calls to conv::pNd(...) for both yakl and kokkos
+# impls for views/arrays that are not getting equivalent results.
+# This is an easy way to quickly isolate problems.
+
+# Remove previous temp files
+/bin/rm JGF*
+
+# Build
+../cmakescript.sh -DRRTMGP_ENABLE_KOKKOS=On -DCMAKE_BUILD_TYPE=Debug
+
+make -j16
```

Comment:
Is there a reason to repeat make commands several times (also below)?

Reply:
There was a race in the f90 builds (now fixed).